### ISSUE TITLE
UIA_ Enums should be signed type

### ISSUE THREAD

**Issue by lonitra** [microsoft/win32metadata#1723]
In the metadata, `UIA_` enums are all defined as `uint` enums. According to UIAutomationClient.idl these enums should be a signed type and Accessibility tools such as Narrator will be expecting these values to be signed as this is what is documented in UIAutomationClient.h
